test/html/HtmlConverterTest.php

```diff
@@ -1,14 +1,24 @@
 <?php
 require_once 'src/wovnio/html/HtmlConverter.php';
 require_once 'src/wovnio/html/HtmlReplaceMarker.php';
-
+require_once 'src/wovnio/wovnphp/Utils.php';
+require_once 'src/wovnio/wovnphp/Headers.php';
 require_once 'src/wovnio/modified_vendor/simple_html_dom.php';
 
 use Wovnio\Html\HtmlConverter;
+use Wovnio\Wovnphp\Url;
```

Comment:
it's strange that we have no `require_once` for Url but still able to use it.

Reply:
Probably because it is required in another file that is require here

Reply:
Yes, you're right. I tried remove all require_once and tests still pass